Segments/Kesmai.mapproj

```diff
@@ -99692,16 +99693,16 @@ return orc;
     
     wizard.Attacks = new CreatureAttackCollection()
     {
-        new CreatureBasicAttack(8)
+        new CreatureBasicAttack(9)
```

Comment:
Wizards and thaums having the same base attack as fighters seem weird. Is this intended.